src/misc/Hooks.h

```diff
@@ -30,4 +30,5 @@ DEFINE_HOOK(GfxInit, void());
 DEFINE_HOOK(ExitGame, void());
 DEFINE_HOOK(LoadFile, void(uint32_t fileNum));
 DEFINE_HOOK(DeleteFile, void(uint32_t fileNum));
+DEFINE_HOOK(CVarInit, void());
```

Comment:
The purpose of this hook is to allow clients to add their CVars after the CVar system has been set up, but before the config file has been loaded and the saved CVars are applied